### DISCUSSION TITLE
student benefits verification

### DISCUSSION THREAD

**Discussion by shajan-santosh2** [community/community#118932] [GitHub Education]
### Hi everyone,

 Can anyone give the exact duration for the github student verification
![Screenshot (11)](https://github.com/community/community/assets/166675095/7c8c10bb-1b65-42ce-98b0-6972fdca5436)

**Comment by samus-aran** [comment]
Hi @shajan-santosh2 

We've updated our [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352) discussion. It's packed with valuable information that might just have what you're looking for. I've included the section that I feel would best assist.

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

Remember to return to your discussion and reply if you need more assistance or mark it as solved 🎉 This will assist others in the community to finding answers more quickly 👍
